### PR TITLE
Remove a few warnings

### PR DESCRIPTION
**Pull request by MillerApps**
A start on removing warnings from the project.

### CODE REVIEW

File: Trust.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3384,13 +3384,15 @@
 				CLANG_WARN_BOOL_CONVERSION = YES;
 				CLANG_WARN_COMMA = YES;
 				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = YES;
```
Comment (vikmeup) [inline Trust.xcodeproj/project.pbxproj]: I think we need this

File: Trust.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -3384,13 +3384,15 @@
...
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_EMPTY_BODY = YES;
 				CLANG_WARN_ENUM_CONVERSION = YES;
 				CLANG_WARN_INFINITE_RECURSION = YES;
 				CLANG_WARN_INT_CONVERSION = YES;
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
```
Comment (vikmeup) [inline Trust.xcodeproj/project.pbxproj]: sane here

File: Trust.xcodeproj/project.pbxproj (hunk 1 above)
Comment (MillerApps) [reply Trust.xcodeproj/project.pbxproj]: I assume you meant we don't need this. If so I have made the appropriate changes.

File: Trust.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -3384,13 +3384,15 @@
...
 				CLANG_WARN_CONSTANT_CONVERSION = YES;
+				CLANG_WARN_DEPRECATED_OBJC_IMPLEMENTATIONS = NO;
 				CLANG_WARN_DIRECT_OBJC_ISA_USAGE = YES_ERROR;
 				CLANG_WARN_DOCUMENTATION_COMMENTS = YES;
 				CLANG_WARN_EMPTY_BODY = YES;
 				CLANG_WARN_ENUM_CONVERSION = YES;
 				CLANG_WARN_INFINITE_RECURSION = YES;
 				CLANG_WARN_INT_CONVERSION = YES;
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES;
+				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = NO;
```
Comment (vikmeup) [inline Trust.xcodeproj/project.pbxproj]: Are these changes added by Xcode automatically?

Comment (MillerApps) [reply Trust.xcodeproj/project.pbxproj]: Yes.

**Pull request merged**